gatsby-config.js

```diff
@@ -69,17 +69,11 @@ module.exports = {
       },
     },
     {
-      resolve: `gatsby-plugin-prefetch-google-fonts`,
+      resolve: `gatsby-plugin-google-fonts`,
       options: {
         fonts: [
-          {
-            family: "IBM Plex Mono",
-            variants: [600],
-          },
-          {
-            family: "Scope One",
-            variants: [400],
-          },
+          "IBM Plex Mono\:600",
+          "Scope One\:400",
```

Comment:
I would be tempted to put `display: 'swap'` in here for a little faster site. If we did that, it would probably be good to do to the other self-hosted fonts we are using too.

Reply:
Good call @tjdunklee , I'll update this.